### DISCUSSION TITLE
How to disable the startup animation

### DISCUSSION THREAD

**Discussion by Kommynct** [hyprwm/Hyprland#6866] [Q&A] [answered]
Hyprland does an animation when launched from TTY on my machine, however, because my machine has just started, it is an extremely slow animation that looks buggy, i'd like to just disable it.

it starts with swaylock if that matters, not sure.

**Comment by verziet** [comment, marked as answer]
Hi,
how i managed to do it is disable animations in the `hyprland.conf`, then I wait and change them back to `true`  with `exec-once = sleep 1 && hyprctl keyword animations:enabled true`

**Reply by Kommynct** [reply]
Good but it should probably be exec instead of exec-once since that'll mean the animations will be disabled if you do a hyprctl reload

Thank you!
